exchanges/mock/recording.go

```diff
@@ -30,7 +33,8 @@ type HTTPResponse struct {
 
 // HTTPRecord will record the request and response to a default JSON file for
 // mocking purposes
-func HTTPRecord(res *http.Response, service string, respContents []byte) error {
+// mockDataSliceLimit defaults to 5
+func HTTPRecord(res *http.Response, service string, respContents []byte, mockDataSliceLimit int) error {
```

Comment:
If you could write a test and have it cover this function in a predictable way,  I'd really appreciate it.
If in the process of writing a test, you change this large function into smaller, tested pieces then that's also cool.

Reply:
This has not been addressed

Reply:
Hello, I wish to inform you I was WRONG and it has been addressed. Thank you